mldsa/sign.h

```diff
@@ -30,6 +39,9 @@
  *                               bytes)
  *
  * Returns 0 (success) or -1 (PCT failure)
+ *
+ * Specification: Implements @[FIPS204 Algorithm 6 ML-DSA.KeyGen_internal]
```

Comment:
```suggestion
 * Specification: Implements @[FIPS204, Algorithm 6 (ML-DSA.KeyGen_internal)]
```

same for all other algorithms

Reply:
Fixed for all algorithm.